samcli/hook_packages/terraform/hooks/prepare/hook.py

```diff
@@ -1451,7 +1455,7 @@ def _format_makefile_recipe(rule_string: str) -> str:
     str
         The formatted target rule
     """
-    return f"\t{rule_string}{os.linesep}"
+    return f"\t{rule_string}\n"
```

Comment:
make our generated make file in a unix format, so it can be executed in all Oss and also in container from a windows machine